_docs/operations/storage-host-config.md

```diff
@@ -1,43 +1,64 @@
 ---
 layout: guide
-title: StorageOS Docs - Extend available storage
+title: StorageOS Docs - Storage configuration
 anchor: operations
-module: operations/extend-storage
+module: operations/storage-host-config
 ---
 
-# Extend available storage
+# Storage host configuration
 
-StorageOS uses the storage available in the nodes where it is installed to present as available for
-volumes. That space is selected from the mount point of `/var/lib/storageos/data` on the host. It is
-recommended to use specific disk devices only for StorageOS.
+StorageOS uses the storage available in the nodes where it is installed to
+present as available for volumes.
 
-StorageOS needs a filesystem formatted and mounted to `/var/lib/storageos/data`. It is possible to
-use different disks or partitions to expand StorageOS' available space. By default, the directory
-`/var/lib/storageos/data/dev1` will be created when volumes are used. However, it is possible to
-shard the data files by creating more directories alike. StorageOS will save data in any directory
-that follows the name `dev[0-9]+`, such as `/var/lib/storageos/data/dev2` or
-`/var/lib/storageos/data/dev5`. This functionality enables operators to mount different devices into
-devX directories and StorageOS will recognise them as available storage automatically.
+It is recommended to isolate the `/var/lib/storageos` directory from the root
+filesystem. Default installations of StorageOS will use `/` as a data store. In
+the event that the `/` partition is filled, the host OS can malfunction.
+Therefore, it is best to mount a different device or partition for the
+StorageOS directory. StorageOS is agnostic to the type of filesystem mounted in
+`/var/lib/storageos`.
 
-These are 2 possible options to expand the available disk space for StorageOS to allocate (combinations of
-these options are also possible):
+In addition, StorageOS can extend the storage available to StorageOS volumes.
+It is possible to use different disks or partitions to expand StorageOS’
+available space. That space is selected from the mount point of
+`/var/lib/storageos/data` on the host. It is recommended to use specific disk
+devices only for StorageOS data dir.
 
-1. Mount filesystem in `/var/lib/storageos/data/devX` (available on next release)
+
+## Extend available storage
+
+StorageOS needs a filesystem formatted and mounted to
+`/var/lib/storageos/data`. It is possible to use different disks or partitions
+to expand StorageOS' available space. By default, the directory
+`/var/lib/storageos/data/dev1` will be created when volumes are used. However,
+it is possible to shard the data files by creating more directories. StorageOS
+will save data in any directory that follows the name `dev[0-9]+`, such as
+`/var/lib/storageos/data/dev2` or `/var/lib/storageos/data/dev5`. This functionality 
+enables operators to mount different devices into devX directories and StorageOS
+will recognise them as available storage automatically.
```

Comment:
I find this paragraph pretty confusing. If different devices are mounted to say ../data/dev2 and ../data/dev3 when do those directories get used? Does all data get written across dev1, dev2 and dev3?

Reply:
done